src/lambdas/get-env.lambda.ts

```diff
@@ -13,21 +13,38 @@ export enum SupportedEnvAttr {
   OpenAiKey = `OPEN_AI_KEY`,
 }
 
-export const getEnvLambda = (
-  attr: SupportedEnvAttr,
-  defaultValue?: string | number,
-): undefined | string => {
-  const envData = process.env[attr]
-  if (envData) return envData
-  if (!defaultValue) return undefined
-  return defaultValue.toString()
+export const envLambda = {
+  get: (
+    attr: SupportedEnvAttr,
+    defaultValue?: string | number,
+  ): undefined | string => {
+    const envData = process.env[attr]
+    if (envData) return envData
+    if (!defaultValue) return undefined
+    return defaultValue.toString()
+  },
+  isProductionEnv: () =>
+    envLambda.get(SupportedEnvAttr.StrictlyEnv) === StrictlyEnv.ProductMode,
+  isLocalMode: () =>
+    envLambda.get(SupportedEnvAttr.StrictlyEnv, StrictlyEnv.DefaultMode) ===
+    StrictlyEnv.LocalMode,
+  isChatGptAllowed: () => {
+    if (!envLambda.isLocalMode()) return false
+    return (
+      envLambda.get(SupportedEnvAttr.StrictlyAllowChatGtp) ===
+      StrictlyAllowChatGtp.AllowChatGpt
+    )
+  },
 }
 
-export enum StrictlyEnv {
+// ! Do not export Strictly imports below. They are only for this file.
```

Comment:
It is a good rule that strict env cannot be exported. Must write a great comments for them though! seems like random developers do not understand why.

Reply:
handled.